portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,32 @@ function openLinkedInUrl() {
 function openGitHubUrl() { 
     window.open("https://github.com/RossJ24","_blank");
 }
+/**
+ * Fetches a greeting with my name from the server.
+ */
+async function getGreeting(){
+ let res = await fetch('/data');
+ let text = await res.text();
+ addGreetingToDOM(text);
+}
+
+/**
+ * Fetches greetings with my name from the server.
+ */
+async function getGreetings(){
+ let res = await fetch('/data');
+ let list = await res.json();
+ let text ='';
+ for(obj of list){
+    text += obj+"\n";
+ }
+ addGreetingToDOM(text);
+}
```

Comment:
style guide suggests that a { is on same line with a space separator. For operands / operators, there are spaces between them (including equality symbols).

Also, format two space indentation.